tempesta_fw/cache.c

```diff
@@ -1144,11 +1144,12 @@ cache_req_process_node(TfwHttpReq *req, unsigned long key,
 	TfwHttpResp *resp = NULL;
 	TDB *db = node_db();
 	TdbIter iter;
+	time_t is_live = tfw_cache_entry_is_live(req, ce);
```

Comment:
This is wrong.
`ce` is not set until later, when the record is found in the cache database. See `tfw_cache_dbce_get()`.